Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/GenericComboBoxCtrl.inl

```diff
@@ -18,6 +18,10 @@ AZ_POP_DISABLE_WARNING
 #include <AzToolsFramework/UI/PropertyEditor/PropertyQTConstants.h>
 #include <AzToolsFramework/UI/PropertyEditor/DHQComboBox.hxx>
 
+
+#pragma optimize("", off)
```

Comment:
Was this just for debugging?

Reply:
Hey yes, sorry I should've changed it to draft I was still working on this!